src/components/analytics-banner/AnalyticsBanner.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import Button from '../button/Button'
+
+const AnalyticsBanner = ({ className, label, yesLabel, onYes }) => {
+  return (
```

Comment:
I thought this was a new file but it looks like it's just a copy of `askToEnable`. Awesome.. thanks for the rename